pom.xml

```diff
@@ -49,8 +50,8 @@
     <maven-site-plugin.version>3.12.1</maven-site-plugin.version>
     <maven-surefire-plugin.version>3.3.1</maven-surefire-plugin.version>
     <spotless-maven-plugin.version>2.43.0</spotless-maven-plugin.version>
-    <kubernetes-client.version>6.13.1</kubernetes-client.version>
-    <okio.version>1.17.6</okio.version>
+    <kubernetes-client.version>6.13.4</kubernetes-client.version>
+    <okio.version>3.9.1</okio.version>
```

Comment:
This is not necessarily compatible. I'd recommend keeping it at 1.17.6 for now.

Reply:
Removed all dependecy bumps from this PR, as we had #51 for that